library/vault

```diff
@@ -8,4 +8,5 @@
 0.8.0: git://github.com/hashicorp/docker-vault@d1cbdce2b531e9223356efb14e828afa556e1f70 0.8.0
 0.8.1: git://github.com/hashicorp/docker-vault@9ddc187d94f68f2572e4d3b1e733c98c661d628b 0.8.1
 0.8.2: git://github.com/hashicorp/docker-vault@0cbe9598ea742e5227a20012332400b0f96a0327 0.8.2
```

Comment:
Any particular reason to keep these older versions listed here?  Removing them here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.) :+1: